subsys/bluetooth/host/gatt.c

```diff
@@ -2068,6 +2069,27 @@ static struct net_buf *nfy_mult[CONFIG_BT_MAX_CONN];
 static int gatt_notify_mult_send(struct bt_conn *conn, struct net_buf **buf)
 {
 	int ret;
+	uint8_t *pdu = (*buf)->data;
```

Comment:
Given the many dereferences on `buf` (i.e. `*buf`), may be worth storing that as a local variable too.

Reply:
just changed buf's type instead. The only reason for the double-ptr was to set it to NULL at the end it seems.